testing/custom/test.sh

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env bash
+
+exec echo "Hello World"
```

Comment:
Is this file commited on purpose?

Reply:
it's here for `/test custom` to work, same as this commit https://github.com/openshift-psap/ci-artifacts/pull/255/commits/6311ca474d12282d4e7b2054190812afa723862d :#

not sure if I should remove it or not :/